schemas/list-provisioners-response.yml

```diff
@@ -41,6 +41,60 @@ properties:
             Date and time where the provisioner was last seen active
           type:         string
           format:       date-time
+        actions:
+          type:         array
+          items:
+            type:       object
+            title:      Actions
+            description: |
+              Actions are defined on the provisioner. When an action is triggered,
+              it will use the `url` and `context` property to construct the URL and
+              perform a `POST` request. Note that the request will be signed with the
+              user's Taskcluster credentials.
+            properties:
+              label:
+                title:  "Label"
+                description: |
+                  Short names for things like logging/error messages.
```

Comment:
Is this unique?  Is this the one I might match on if I want to tie a specifi action to a button button in Tools?  For task-graph actions we have [`name` for this purpose](https://docs.taskcluster.net/manual/using/actions/spec#metadata)  and some [guidelines for UIs](https://docs.taskcluster.net/manual/using/actions/ui#specialized-triggering).

Reply:
Yes it is unique. I  agree that we should use `name` instead of `label` to respect the UI guidelines.